### PR TITLE
chore(issue templete): update issue template

### PR DESCRIPTION
**Pull request by tigercl**
**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked.:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

## More information

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -1,26 +1,28 @@
 ---
 name: Bug Report
-about: Create a report to help us improve
+about: Report a bug encountered while operating EMQ X
```
Comment (zmstone) [inline .github/ISSUE_TEMPLATE/bug-report.md:3]:
```suggestion
about: Report a bug of EMQ X
```

File: .github/ISSUE_TEMPLATE/support-request.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+name: Support Request
+about: Support request or question relating to EMQ X
```
Comment (zmstone) [inline .github/ISSUE_TEMPLATE/support-request.md:3]:
```suggestion
about: Support request or question
```

File: .github/ISSUE_TEMPLATE/enhancement-request.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name: Enhancement Request
+about: Suggest an enhancement to the EMQ X project
```
Comment (zmstone) [inline .github/ISSUE_TEMPLATE/enhancement-request.md:3]:
```suggestion
about: Suggest an enhancement
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -1,26 +1,28 @@
...
-<!-- 请使用英文描述问题 -->
+<!-- Please use this template while reporting a bug and provide as much info 
+as possible so that we can support you more efficiently. Thanks!-->
 
-**Environment**:
+#### What happened:
+
+#### What you expected to happen:
+
+#### How to reproduce it (as minimally and precisely as possible):
```
Comment (zmstone) [inline .github/ISSUE_TEMPLATE/bug-report.md:17]:
```suggestion
#### How to reproduce it (prefer minimal and precise steps):
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 5]
```diff
@@ -1,26 +1,28 @@
...
 
-<!-- Please use this template while reporting a bug and provide as much info as possible. Thanks!-->
-<!-- 请使用英文描述问题 -->
+<!-- Please use this template while reporting a bug and provide as much info 
+as possible so that we can support you more efficiently. Thanks!-->
 
-**Environment**:
+#### What happened:
+
+#### What you expected to happen:
```
Comment (zmstone) [inline .github/ISSUE_TEMPLATE/bug-report.md:15]:
```suggestion
#### Expected behaviour
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 6]
```diff
@@ -1,26 +1,28 @@
...
+
+#### How to reproduce it (as minimally and precisely as possible):
+
+#### Anything else we need to know:
```
Comment (zmstone) [inline .github/ISSUE_TEMPLATE/bug-report.md:19]:
```suggestion
#### More information to help understand/troubleshoot
```

File: .github/ISSUE_TEMPLATE/support-request.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+#### Support Request or Question
+
+#### Environment:
+
+- EMQ X version (e.g. `emqx_ctl status`):
+- Hardware configuration (e.g. `lscpu`):
+- OS (e.g. `cat /etc/os-release`):
+- Kernel (e.g. `uname -a`):
+- Erlang/OTP version (in case you build EMQ X from source code):
```
Comment (zmstone) [inline .github/ISSUE_TEMPLATE/support-request.md:20]: - Provide the content in this file: `emqx/releases/<version>/BUILT_ON`

Comment (zmstone) [reply .github/ISSUE_TEMPLATE/support-request.md:20]: for emqx v4.3 or newer

**Comment by HJianBo** [comment]
This looks like a very early PR, what's its status now?

**Review by terry-xiaoyu — APPROVED** [review]

**Pull request closed without merging**